inputs/sourmash_databases/sourmash-database-info.csv

```diff
@@ -0,0 +1,16 @@
+﻿lineage,ksize,osf_hash

```

Comment:
I was thinking of a dictionary, but this works too I guess!

Reply:
ya -- bc R was my first language, I think in dataframes instead of in dictionaries by default. I figured dictionary would be the pythonic way to do it, but it would have taken me an extra hour, so I just did it this way 🤷‍♀️